config/files/usr/etc/NetworkManager/conf.d/rand_mac.conf

```diff
@@ -0,0 +1,9 @@
+[device-mac-randomization]
+# "yes" is already the default for scanning
+wifi.scan-rand-mac-address=yes
+ 
+[connection-mac-randomization]
+# Randomize MAC for every ethernet connection
+ethernet.cloned-mac-address=random
+# Generate a random MAC for each WiFi and associate the two permanently.
+wifi.cloned-mac-address=stable
```

Comment:
This makes the mac address random per-network, not per-connection?

Reply:
Maybe i misunderstood the discussion in #72. I thought the problem with random per connection is WIFI only. Should it be "per Network" on ethernet too?

Reply:
I'm just trying to understand what the config does. I'm not questioning it.

Reply:
`stable` means that you will use the same MAC address every time you connect to that network.

Reply:
Thinking about it now `stable` randomization makes sense for wired connections too and having only one setting reduces the complexity too

Reply:
Agreed.